weather_dl/setup.py

```diff
@@ -32,7 +32,7 @@
 setup(
     name='download_pipeline',
     packages=find_packages(),
-    version='0.1.2',
+    version='0.1.3',
```

Comment:
Please confirm this is the right version with the main branch.

Reply:
The version of weather_dl is in sync with main branch.